### PR TITLE
Fix frontend Dockerfile not starting

### PR DESCRIPTION
**Pull request by Clechay**
Container was failing because it was using node_modules from developer's env instead of node_modules created during image build.

### CODE REVIEW

File: frontend/Dockerfile [hunk 1]
```diff
@@ -1,12 +1,14 @@
-FROM node:12.2.0-stretch-slim
+FROM node:current-stretch-slim
```
Comment (mkochanowski) [inline frontend/Dockerfile]: For Docker images, using anything other than a specific, carefully selected and tested tagged image is considered to be a bad practice - we want to set up Continous Delivery, so when it comes to builds, we have to minimize the risk and disallow third parties from introducing updates that may unexpectedly break backwards compatiblity. This is even more true for the *Dockerfile.production* image.

**TLDR** version numbers should be bumped manually.

```suggestion
FROM node:12.6.0-stretch-slim
```

File: frontend/Dockerfile.production [hunk 2]
```diff
@@ -1,8 +1,14 @@
-FROM node:11.12.0-alpine
+FROM node:current-stretch-slim
```
Comment (mkochanowski) [inline frontend/Dockerfile.production]:
```suggestion
FROM node:12.6.0-stretch-slim
```

File: frontend/Dockerfile [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
 
-COPY . /app 
-WORKDIR /app 
-RUN yarn install
+# RUN /bin/bash -c mount --bind /maisie_node_modules /app/node_modules
+
+RUN yarn global add create-react-app
+RUN yarn global add react-scripts
 
-RUN /bin/bash -c mount --bind /maisie_node_modules /app/node_modules
```
Comment (mkochanowski) [inline frontend/Dockerfile:9]: Getting rid of this command might actually break the container when the volume is bound to a filesystem hosted on Windows. I will try to check whether this statements still holds true.

**Review by mkochanowski — CHANGES_REQUESTED** [review]

File: frontend/Dockerfile (hunk 3 above)
Comment (Clechay) [reply frontend/Dockerfile:9]: fixed by adding node_modules to .dockerignore

**Review by mkochanowski — APPROVED** [review]
